pandas/tests/io/formats/test_to_latex.py

```diff
@@ -448,6 +448,169 @@ def test_to_latex_longtable(self, frame):
         with3columns_result = df.to_latex(index=False, longtable=True)
         assert r"\multicolumn{3}" in with3columns_result
 
+    def test_to_latex_caption_label(self, frame):
+        the_caption = 'a table in a \\texttt{table/tabular} environment'
+        the_label = 'tab:table_tabular'
+
+        frame.to_latex(
+            caption=the_caption,
+            label=the_label
+        )
+
+        df = DataFrame({'a': [1, 2], 'b': ['b1', 'b2']})
```

Comment:
I see multiple tests here.  In the interests of modularity, let's break up into multiple test methods.

Reply:
No problem but it would not be consistent with the other test methods in `test_to_latex.py`; would that be an issue?

Reply:
@gfyoung no problem but it would not be consistent with the other test methods in `test_to_latex.py`; would that be an issue?